### ISSUE TITLE
Modal Content is destroyed when datepicker .hide() is called

### ISSUE THREAD

**Issue by manuakasam** [uxsolutions/bootstrap-datepicker#2330]
### Expected behaviour
Modal  window should remain untouched

### Actual behaviour
Modal Content is destroyed - Modal Blackscreen remains

### Datepicker version used

ex. 1.7.0. (previously same behavior in 1.5.0)

## Debug

I'm calling a Form via ajax into a modal dialog (BS3). Inside said dialog I use the datepicker (date-provide=datepicker). When selecting a date the hide() function is called and will also remove the modal dialog, but the grey/black backdrop remains.

I've tracked this down to this line:

`
        hide: function(){
            if (this.isInline || !this.picker.is(':visible'))
                return this;
            this.focusDate = null;
            this.picker.hide().detach();
            this._detachSecondaryEvents();
            this.setViewMode(this.o.startView);

            if (this.o.forceParse && this.inputField.val())
                this.setValue();
//             this._trigger('hide'); // <-- This is the culprit
            return this;
        },`

Is anyone able to explain what this line is used for? Because commenting out this line I can't find any errors and things appear to just work fine whilst also fixing the issue I'm experiencing.
